tests/system/providers/snowflake/example_snowflake.py

```diff
@@ -50,23 +51,25 @@
     catchup=False,
 ) as dag:
     # [START howto_operator_snowflake]
-    snowflake_op_sql_str = SnowflakeOperator(task_id="snowflake_op_sql_str", sql=CREATE_TABLE_SQL_STRING)
+    snowflake_op_sql_str = SQLExecuteQueryOperator(
```

Comment:
Don't we need to change the `snowflake_conn_id`  key in DAG `default_args`
`default_args={"snowflake_conn_id": SNOWFLAKE_CONN_ID},`

Reply:
yes thanks for the catch @dirrao

Reply:
But should not we remove the "snowflake_conn_id" completely ? Anything preventing it?

Reply:
@potiuk `SnowflakeSqlApiOperator` is an operator to support multiple SQL statements execution sequentially. so it has inherited from` SQLExecuteQueryOperator` to support this behaviour and accepts the `snowflake_conn_id` as its param. that's why I retained both `conn_id` and `snowflake_conn_id` params in this example. Do you think we can change  `snowflake_conn_id` param in `SnowflakeSqlApiOperator` to `conn_id` itself?

https://github.com/apache/airflow/blob/d5f81a4e2de0d4236cffcf2e2d3c682b4c6ec355/airflow/providers/snowflake/operators/snowflake.py#L391-L494

Reply:
right